src/DucksNet.WebUI/Pages/Models/CreateEmployeeModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace DucksNet.WebUI.Pages.Models;
+
+public class CreateEmployeeModel
+{
+    [Required]
+    [StringLength(36, MinimumLength = 36, ErrorMessage = "Guid must be 36 characters long.")]
+    public string? IdOffice { get; set; }
+    [Required]
+    [StringLength(120, MinimumLength = 5, ErrorMessage = "Surname must be at least 5 characters long.")]
+    public string? Surname { get; set; }
+    [Required]
+    [StringLength(120, MinimumLength = 5, ErrorMessage = "First Name must be at least 5 characters long.")]
```

Comment:
5 characters is a bit restrictive for name and surname. for example a `John Doe` could not be created in this system.